wsclient/src/main/java/org/hpccsystems/ws/client/HPCCFileSprayClient.java

```diff
@@ -718,7 +722,74 @@ public boolean uploadFileLocalDropZone(File file) throws Exception
 
         return uploadFile(file, fetchLocalDropZones[0]);
     }
+    public boolean upLoadLandingZone(File uploadFile,DropZone dropZone) {
+  	
+    	String boundary = Utils.createBoundary();
+    	
+    	URLConnection fileUploadConnection = null;
+        URL fileUploadURL = null;
+        String FILESPRAYWSDLURI     = "/FileSpray";
```

Comment:
We might as well ensure File is valid right up front before setting up the connection